lib/systems/examples.nix

```diff
@@ -93,6 +93,10 @@ rec {
     config = "mipsel-unknown-linux-gnu";
   } // platforms.fuloong2f_n32;
 
+  mips64el-linux-gnuabin32      = platforms.mips64el-linux-gnuabin32;
+  mips64el-linux-gnuabi64       = platforms.mips64el-linux-gnuabi64  // { rustc.config = "mips64el-unknown-linux-gnuabi64"; };
```

Comment:
what happens if you don't have the `rustc.config` like that? I would just skip that for now if you are not sure.

Reply:
If you don't have it, rust won't build!

With it, and #162070 (which the reviewer marked-as-draft pending merge of this PR), rust bootstraps very nicely on mips64el.

See also: https://doc.rust-lang.org/rustc/platform-support.html (Tier 2).

Unfortunately rustc does not provide a bootstrap tarball for `mips64el-linux-gnuabin32`.  I think it can cross-compile to it as a buildPlatform however (need to double-check this).

Reply:
I have pushed an additional commit with a one-line comment explaining why gnuabi64 has a rustc.config but gnuabin32 does not.

If you prefer, I can move these changes into https://github.com/NixOS/nixpkgs/pull/162070

Reply:
Yes I think it is better we leave those for that, sorry.

Reply:
Now that `platforms.nix` doesn't contain `config` or similar, I think you will want to put that in this file, right? That's why I mentioned *moving* them over.